### ISSUE TITLE
Safari: Fix focus issues 

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5041]
I started to open tickets for what does not work on safari, but there's too many. 
So I'm opening this ticket to fix all the issues, and mention what works.

Only these work:
Lang/Currency
Search Bar
And other little things like "Ship to address" On checkout shipping address page.

**Comment by developpeurweb** [comment]
@KateChuen 
After some research it turns out Safari needs to be expressly enabled to extend its accessibility feature, therefore in order to retest for Safari issues, let's please enable the following:

1. Theme Calydon
2. Safari > Preferences > Advanced > Press Tab to highlight each item on a webpage

![Screen Shot 2019-11-26 at 4 08 17 PM](https://user-images.githubusercontent.com/15113219/69684604-3b50c580-1087-11ea-93ef-59fc5a0eb154.png)

**Comment by developpeurweb** [comment]
Most of the issues fixed in ticket GH-6052
